src/libraries/System.Net.Mail/src/System/Net/Mail/MailAddress.cs

```diff
@@ -63,6 +63,56 @@ public MailAddress(string address, string displayName) : this(address, displayNa
         //
         // A FormatException will be thrown if any of the components in 'address' are invalid.
         public MailAddress(string address, string displayName, Encoding displayNameEncoding)
+        {
+            bool parseSuccess = TryParse(address, displayName, displayNameEncoding, out (string displayName, string user, string host, Encoding displayNameEncoding) parsedData, throwExceptionIfFail: true);
```

Comment:
Could you split the TryParse call so that the ```throwExceptionIfFail: true``` argument is visible? The argument value is here IMHO pretty fundamental for the behavior but it's so far on the line, that I can't see it on my 26'' screen.

Reply:
Sure

Reply:
Splitted in three lines.